recipe/meta.yaml

```
@@ -1,44 +1,42 @@
 {% set name = "PasteDeploy" %}
-{% set version = "2.0.1" %}
+{% set version = "3.1.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d423fb9d51fdcf853aa4ff43ac7ec469b643ea19590f67488122d6d0d772350a
+  sha256: 9ddbaf152f8095438a9fe81f82c78a6714b92ae8e066bed418b6a7ff6a095a95
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  skip: true
```

```suggestion
  skip: true  # [py<37] 
```
`skip: true` skips all platforms.
We skip unsupported Python versions, see https://github.com/Pylons/pastedeploy/blob/3.1/setup.cfg#L44

---

Added